Koronavilkku/Koronavilkku/ViewController/OnboardingViewController.swift

```diff
@@ -15,6 +15,7 @@ class OnboardingViewController: UINavigationController {
         // Note that the steps and StepId values must be in the same order.
         Step(id: .intro,
              buttonTitle: Translation.ButtonNext.localized,
+             showLanguageSelection: true,
              view: StepView(image: UIImage(named: "radar-static")!,
```

Comment:
Not related to this review, but this is actually wrong image resource. See Figma for more details